damnit/gui/table.py

```diff
@@ -175,6 +178,58 @@ def get_static_columns(self):
 
     def get_static_columns_count(self):
         return self._static_columns_widget.count()
+    
+    def contextMenuEvent(self, event):
+        self.menu = QtWidgets.QMenu(self)
+        self.zulip_action = QtWidgets.QAction('Export table to Zulip', self)
+        self.zulip_action.triggered.connect(self.export_selection_to_zulip)
+
+        self.menu.addAction(self.zulip_action)
+        self.menu.popup(QtGui.QCursor.pos())
+
+    def export_selection_to_zulip(self):
+        if not isinstance(self.model()._main_window.zulip_messenger, ZulipMessenger):
+            self.model()._main_window.zulip_messenger = ZulipMessenger(self.model()._main_window)       
+            
+        selected_rows = [r.row() for r in 
+                         self.selectionModel().selectedRows()]
+        bad_columns = ['Proposal', 'comment_id', 'Status']
+        df = self.model()._main_window.data.iloc[selected_rows]
+        bad_columns = bad_columns + self.columns_with_thumbnails(self.model()._data) \
```

Comment:
could you attach pictures of plots to the message if columns with thumbnails are selected?

Reply:
This is already supported since upon double clicking thumbnails, a Canvas instance is created. Meaning that the plot behaves like any other.